pp/sidebarsPp.js

```diff
@@ -2160,7 +2160,11 @@ module.exports = {
         },
         {
           type: 'doc',
-          id: 'integrations/plugin-packs/procedures/hardware-storage-hp-storeonce-restapi'
```

Comment:
if you delete hardware-storage-hp-storeonce-restapi from sidebarspp.js, the old deprecated page with redirection to the new ones won't be accessible?

Reply:
it will be accessible from the pack manager. we don't want to be accessible from the menu.